android/settings.gradle

```diff
@@ -1,6 +1,7 @@
 include ':app-example'
```

Comment:
Also might be worth to change builders in app-example too?

Reply:
Still working on it :)